api-samples/action/demo/index.js

```diff
@@ -123,6 +122,55 @@ document
     showBadgeText();
   });
 
+// --------------------------
+// get/set badge text color
+// --------------------------
+async function showBadgeTextColor() {
+  const color = await chrome.action.getBadgeTextColor({});
+  document.getElementById('current-badge-txt-color').value = JSON.stringify(
+    color,
+    null,
+    0
+  );
+}
+
+showBadgeTextColor();
+
+document
+  .getElementById('set-badge-txt-color-button')
+  .addEventListener('click', async () => {
+    // To show off this method, we must first make sure the badge has text
+    let currentText = await chrome.action.getBadgeText({});
+    if (!currentText) {
+      chrome.action.setBadgeText({ text: 'hello :)' });
```

Comment:
As fun as this is we should probably do something a bit more serious. "Test" or "Demo" maybe?